core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -17,56 +17,99 @@
  * fields.
  */
 
+export class StateAndContentContainer {
```

Comment:
"Container" is a bit redundant and doesn't add too much value to the name since any class is essentially a "container" for some data. I think we can drop it.